### ISSUE TITLE
(info?) I made a tool that compares two local indexes with various metadata formats

### ISSUE THREAD

**Issue by jmrgibson** [ChrisGreenaway/cargo-local-registry#156]
I made a tool that compares to local indexes to handle the case of changing metadata formats so you can audit there were no crates removed.

Maybe we could get a link in the readme for other people that might find this useful? Or I could also merge it into this project if we wanted to add a `check` subcommand or something.

https://github.com/jmrgibson/cargo-local-registry-compare

**Comment by ChrisGreenaway** [comment]
That looks cool. I'd be happy to get that merged into this project, if that's what you'd like to do. Do you have the time to raise a PR to make it happen?

**Comment by jmrgibson** [comment]
Yup, I can clean it up and make a PR. Should I make a second binary in the current Cargo.toml? Or a subcommand of the existing binary?

**Comment by ChrisGreenaway** [comment]
I'd favour a sub-command.
